app/models/settings.rb

```diff
@@ -91,6 +91,11 @@ def password_login_allowed?
     true
   end
 
+  def spam_protection_level
+    level = super.to_sym
+    level == :auto ? :captcha : level
```

Comment:
This `:auto` is the old suspicious only mode, is called `auto` in the DB. We can't change the values in table, they will still be called `:none`, `:auto` and `:captcha`. `:auto` is now deprecated and treated as `:captcha` at all effects. I think it's fine to rename 'Never' and 'Always' to 'None' and 'Captcha'. I'll do it

Reply:
The renaming is good. I think it would be nice to add a comment saying that `:auto` is for backwards compatibility as the "auto" mode is being replaced with "captcha" mode.
Because otherwise, future us (or other folks) will scratch their heads at this line... :roll_eyes:

Reply:
Added the comment in https://github.com/3scale/porta/pull/3645/commits/2ed4366303def28c1be82fbd3812e0b08d130fe7